macos_ether.h

```diff
@@ -0,0 +1,21 @@
+#include <netinet/if_ether.h>
```

Comment:
Have you tried this one?
https://unix.superglobalmegacorp.com/xnu/newsrc/bsd/netinet/ip.h.html

Reply:
Yes, this file contains the `struct ip` (instead of `struct iphdr`) which contains the same attributes but with different name...
So we will keep our home-made struct